suse_migration_services/prechecks/repos.py

```diff
@@ -33,6 +33,11 @@ def remote_repos():
                             no_remote_repos.append(repo_info['baseurl'])
 
     if no_remote_repos:
+        zypper_cmd = 'zypper repos --url'
```

Comment:
I haven't look in more detail, but does this mean only in case there are no remote repos we show the error message ? What if there are 2 remote repos and 1 local repo, will the check apply and warn about the 1 local repo to be an issue ?

Reply:
No, it means _if there are any remote repos, those are mentioned_

```python
for section in repo_section.keys():
                repo_info = dict(config.items(section))
                if repo_info:
                    if not repo_info['baseurl'].startswith('http') and not repo_info['baseurl'].startswith('ftp'):
                        if repo_info['baseurl'] not in no_remote_repos:
                            no_remote_repos.append(repo_info['baseurl'])
```

loop through the repos
check repo is not remote
add it if it is

I can add that case in the tests, in this PR no problem

Reply:
ok, then I just got confused by the name. All fine, thanks